doc/Makefile.am

```diff
@@ -42,17 +42,17 @@ SUFFIXES=.txt .adoc .xml .xml7 .1 .3 .7
 	asciidoc -d manpage -b docbook -f $(srcdir)/asciidoc.conf \
         -afty-rest_version=@PACKAGE_VERSION@ -o$@ $<
 .xml.1:
-	xmlto --skip-validation man $<
```

Comment:
Note for others : As discussed with Jana, this flag was my (failed) experiment to let fty-rest build on CentOS, but the issue (of missing DTDs) should probably be addressed from packaging somehow.

Similar problem was recently discussed for CZMQ community, so maybe an upstream solution will soon appear.